pkgs/tools/security/osv-scanner/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  osv-scanner,
+}:
+buildGoModule rec {
+  pname = "osv-scanner";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-AftJS8IdKc1oDJfDTB/HMLHu7EAVnPGPR61SOlTnC2E=";
+  };
+
+  vendorSha256 = "sha256-HUgzoQuWBRnt8+lCiu9QfO1XR5EMnqVIkrL+nIMf0IA=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=${version}"
+  ];
+
+  # Tests require network connectivity to query https://api.osv.dev.
+  doCheck = false;
+
+  passthru.tests.version = testers.testVersion {
+    package = osv-scanner;
+  };
+
+  meta = with lib; {
+    description = "Vulnerability scanner written in Go which uses the data provided by https://osv.dev";
+    homepage = "https://github.com/google/osv-scanner";
+    changelog = "https://github.com/google/osv-scanner/releases/tag/v${version}";
+    license = licenses.asl20;
+    maintainers = with maintainers; [stehessel];
```

Comment:
if you would like assistance with maintenance, free to add me, otherwise ignore:
```suggestion
    maintainers = with maintainers; [stehessel urandom];
```

Reply:
Sure